### DISCUSSION TITLE
How and when are z-wave JS versions updated?

### DISCUSSION THREAD

**Discussion by webmogul1** [zwave-js/zwave-js-ui#2887] [Q&A] [answered]
I noticed that z-wave JS UI was updated to 8.7.0

But z-wave JS still at 10.3.1 despite it has already been updated to 10.4.0

How and when are z-wave JS versions updated?

**Comment by robertsLando** [comment, marked as answer]
Zwave-js version has been released after the 8.7 update (few hours after). BTW I'm already working on integrating the new features of 10.4.0.

https://github.com/zwave-js/zwave-js-ui/pull/2886
https://github.com/zwave-js/zwave-js-ui/pull/2885
https://github.com/zwave-js/zwave-js-ui/pull/2880

**Comment by webmogul1** [comment]
Awesome! Thanks for letting me know.
